docs/rst/notes/forthcoming_version.rst

```diff
@@ -11,6 +11,9 @@ This release includes the following **mayor changes**:
   - `ddsrouter_core`: C++ library which implements the DDS Router operation and exports the DDS Router C++ API.
   - `ddsrouter_yaml`:  C++ library to parse the DDS Router *yaml* configuration files.
   - `ddsrouter_tool`: DDS Router end-user application.
+* New dynamic topic discovery feature: the user does not need to fully specify over which topics to communicate (i.e.
```

Comment:
New dynamic topic discovery feature to atomatically discover DDS Topics in a DDS network.